tests/e2e/support/commands.js

```diff
@@ -204,7 +209,7 @@ Cypress.Commands.add('subscribeOpponent', (id) => {
         if (jwres.statusCode === 200) {
           return resolve();
         }
-        return reject(new Error('error subscribing'));
+        return reject(new Error(jwres));
```

Comment:
This was how I finally got cypress to display the error in the terminal output. What sucks is I haven't been able to get it to show both a detailed message and the error in the response. Not sure what's the best practice here. Ideally I'd like it to send a specific message indicating where we are in the control flow and also to display the server's response so we know the status code and message